### ISSUE TITLE
Dermal perivascular lymphocytic vasculitis

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9599]
**Preferred term label:**
Dermal perivascular lymphocytic vasculitis

papillary dermal edema
and an interface dermatitis with keratinocyte
necrosis

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
614415 
PMID: 21204240  1/1

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000263
- label: Papillary dermal edema
- parents: [Abnormal dermis morphology (HP:0034924)](https://hpo.jax.org/app/browse/term/HP:0034924)
- pmid: PMID:20377671
- def: Swelling due to fluid acculuation of the papillary dermis.
- comment: The dermis has two major layers: the upper papillary dermis and the lower reticular dermis. The papillary dermis: The papillary dermis consists of bundles of collagen fibrils and oxytalan elastic fibers, ans is separated from the lower reticular dermis by a subpapillary plexus.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9599
- synonyms:
